EasyCommands/Common/Utilities.cs

```diff
@@ -30,10 +30,16 @@ partial class Program {
         public static List<T> NewList<T>(params T[] elements) => new List<T>(elements);
         public static Dictionary<T, U> NewDictionary<T, U>(params KeyValuePair<T, U>[] elements) => elements.ToDictionary(e => e.Key, e => e.Value);
         public static KeyValuePair<T, U> KeyValuePair<T, U>(T key, U value) => new KeyValuePair<T, U>(key, value);
-        public static Variable EmptyList() => GetStaticVariable(NewKeyedList());
+
+        //Utilities for constructing enumerables
+        public static IEnumerable<int> Range(int start, int end) => Enumerable.Range(start, end);
+        public static IEnumerable<T> Empty<T>() => Enumerable.Empty<T>();
+        public static IEnumerable<T> Once<T>(T element) => Enumerable.Repeat(element, 1);
```

Comment:
not sure I understand the value add from these ones.  How much better is the performance?  These two cost around 100 chars.

nit: can we remove ```Enumerable.```, as we did elsewhere?

Reply:
The main purpose of these functions is to better signal intent and avoidance of allocations. Some but not all usage can be replaced with `NewList`. All in all adding these functions costs only around 20-30 chars but increases readability and removes allocations.

Instead of `Enumerable.` these functions are called, so no